recipes/pyphysim/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "pyphysim" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0df90c4d8031503d69d2cbd13717daf34cab8295a73aaecdb70e08441cdeaaae
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or py<37]
```

Shouldn't this be what you want? The error in CI is about a line that is only working for Python 3.8+
```suggestion
  skip: True  # [win or py<38]
```